appeals/web/src/server/appeals/appeal-details/costs/costs.controller.js

```diff
@@ -466,12 +485,33 @@ export const renderDecisionCheckAndConfirm = async (request, response) => {
 		return response.status(500).render('app/500.njk');
 	}
 
+	let documentVersion;
+	let documentFileName;
+
+	if (documentId) {
+		const fileInfo = await getFileInfo(request.apiClient, currentAppeal.appealId, documentId);
+
+		if (!fileInfo) {
+			console.log('documentId not valid, 404');
+			return response.status(404).render('app/404');
+		}
+
+		if (!('latestDocumentVersion' in fileInfo) || !('name' in fileInfo)) {
+			return response.status(500).render('app/500.njk');
+		}
+
+		documentVersion = fileInfo.latestDocumentVersion.version + 1;
+		documentFileName = fileInfo.name;
+	}
```

Comment:
Minor, something to consider. I think something like this would be a bit more readable:

```
		const {latestDocumentVersion, name} = fileInfo;

		if (!latestDocumentVersion || !name) {
			return response.status(500).render('app/500.njk');
		}

		documentVersion = latestDocumentVersion?.version + 1;
		documentFileName = name;
```

Reply:
Good suggestion, thanks - updated! Also made me realise I'd left a console.log in, that's been removed too.